tests/test_util.go

```diff
@@ -297,3 +297,69 @@ func removeSecret(cli kubernetes.Interface, secretName, ns string) error {
 	}
 	return nil
 }
+
+func arangoDeploymentHealthy(t *testing.T, deployment *api.ArangoDeployment, k8sClient kubernetes.Interface) {
```

Comment:
To be in line with other `waitUntil...` functions, this function must return an `error` in case something is wrong.
You can (read should) then remove the `t *testing.T` argument.